Spec/Auth.swift

```diff
@@ -2620,104 +2620,84 @@ class Auth : QuickSpec {
                 }
             }
         }
-        
-        describe("Reauth") {
-            // RTC8
-            pending("should use authorise({force: true}) to reauth with a token with a different set of capabilities") {
-                // init ARTRest
-                let restOptions = AblyTests.setupOptions(AblyTests.jsonRestOptions)
-                let rest = ARTRest(options: restOptions)
 
-                // get first token
-                let tokenParams = ARTTokenParams()
-                tokenParams.capability = "{\"wrongchannel\": [\"*\"]}"
-                tokenParams.clientId = "testClientId"
+        describe("Reauth") {
 
-                var firstToken = ""
+            // RTC8
+            it("should use authorise({force: true}) to reauth with a token with a different set of capabilities") {
```

Comment:
Ok, but I'm not sure why we are doing any work on `0.8` now other than critical fixes?